frontend/src/metabase/components/NewItemMenu/NewItemMenu.tsx

```diff
@@ -136,9 +138,9 @@ const NewItemMenu = ({
         title: t`Metric`,
         icon: "metric",
         link: Urls.newQuestion({
-          mode: "notebook",
-          collectionId,
+          mode: "query",
```

Comment:
I don't understand why did we change this to `query`? If you take a look at the question flow, the URL says `/notebook` which seems to reflect the UI state correctly.

Now, compare it to the metric:
![image](https://github.com/metabase/metabase/assets/31325167/d0b2f13b-d1d8-447b-9c0a-c8ef29c618ce)

Unless there was a really good reason for this change, I think we should change it back to the "notebook".

Reply:
> If you take a look at the question flow

We want to follow the model flow.

Reply:
Ok, I was about to correct myself because I've just opened a model and saw that it has a `/query` in the URL.

Admittedly, this is adding new layer of confusion but that has nothing to do with your changes here.
Ignore my comment.